tables/tests/test_all.py

```diff
@@ -11,7 +12,9 @@
 from tables.tests.test_suite import suite, test
 
 
-faulthandler.enable()
+# Give people a way to opt out of enabling faulthandler
+if os.getenv("PYTABLES_DISABLE_FAULTHANDLER", "").lower() not in ("1", "true"):
+    faulthandler.enable()
```

Comment:
U was more thinking to [PYTHONFAULTHANDLER](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONFAULTHANDLER) actually, but the current implementation is also fine for me.

Reply:
Ahh I didn't even know about that env var. Yes it could be done this way but then all CIs that call tests would have to be updated to have this env var, as would devs when testing locally, etc. So probably less useful than an opt-out mechanism I think?

Reply:
I agree
Thanks